src/components/Explore/Map.tsx

```diff
@@ -188,6 +194,7 @@ const ExploreMap = (): JSX.Element => {
         // When map is ready and stationsList change, update the data for `stations` and `clustered-stations`
         const map = mapRef.current;
         if (map && isMapLoaded) {
+            // console.log(stationsList);
```

Comment:
Can you remove the commented-out console logs?

Reply:
yup